### ISSUE TITLE
"az network vhub get-effective-routes" Not working as expected.

### ISSUE THREAD

**Issue by Lochiluk** [Azure/azure-cli#18380]
"az network vhub get-effective-routes" Not working as expected.

lohith@Azure:~$ az network vhub get-effective-routes --resource-type RouteTable --resource-id /subscriptions/4b15d5a8-6a11-4ad3-a6a8-dc34e0a4221b/resourceGroups/VWAN/providers/Microsoft.Network/virtualHubs/EastHUB/hubRouteTables/defaultRoutetable -g VWAN -n EastHUB
(InvalidRequestFormat) Cannot parse the request.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cf41819d-ee6a-8246-8444-a4793de4bab4
* Version Independent ID: cd0448cb-df88-af36-03ce-9ff4a446d8f3
* Content: [az network vhub](https://docs.microsoft.com/en-us/cli/azure/network/vhub?view=azure-cli-latest&source=docs#az_network_vhub_get_effective_routes)
* Content Source: [latest/docs-ref-autogen/network/vhub.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/latest/docs-ref-autogen/network/vhub.yml)
* Service: **virtual-network**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
network

**Comment by kairu-ms** [comment]
Which version of cli and extension are you using? Could you please update cli and extension to the latest version and have a try again. Thanks.

**Comment by Lochiluk** [comment]
The VWAN CLI extension has been updated to 0.2.6 and after testing this with Welly we figured out that the new version (0.2.5 & 0.2.6) of the CLI are having issues for get effective routes for HUB route tables. For time being, we can roll back to 0.2.4 by using the below commands. This should work. (Tested internally)
 
Remove the VWAN extension: az extension remove --name virtual-wan
Add the old version: az extension add --name virtual-wan --version "0.2.4"
 
FYI, this applies for both online(Shell.azure.com) and offline(CMD/PS) versions. (Tested)
 
PGs are working to resolve this issue internally with no ETA as of now.
